plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/PartitionTransforms.java

```diff
@@ -373,6 +380,14 @@ private static int bucketHash(Slice value)
         return Murmur3Hash32.hash(value);
     }
 
+    private static int bucketStringHash(Slice value)
+    {
+        // We need to use slower implementation to match Iceberg
```

Comment:
slower than what? how should i read this comment when encontered in the code (vs in pr review)?